### PR TITLE
style: Removes extra spaces

### PR DESCRIPTION
**Pull request by mohammadamin16**

### CODE REVIEW

File: src/components/PublishLibrary.tsx [hunk 1]
```diff
@@ -409,7 +409,7 @@ const PublishLibrary = ({
               rel="noopener noreferrer"
             >
               {t("publishDialog.noteDescription.link")}
-            </a>{" "}
+            </a>
```
Comment (dwelle) [inline src/components/PublishLibrary.tsx:412]: the space here is needed

File: src/components/LibraryMenuHeaderContent.tsx [hunk 2]
```diff
@@ -106,7 +106,7 @@ export const LibraryDropdownMenuButton: React.FC<{
         <p>
           {t("publishSuccessDialog.content", {
             authorName: publishLibSuccess!.authorName,
-          })}{" "}
+          })}
```
Comment (dwelle) [inline src/components/LibraryMenuHeaderContent.tsx:109]: space is needed here

File: dev-docs/docs/@excalidraw/excalidraw/api/props/render-props.mdx [hunk 3]
```diff
@@ -105,8 +105,8 @@ function App() {
   return (
     <div style={{ height: "500px" }}>
       <button className="custom-button" onClick={() => excalidrawAPI.toggleMenu("customSidebar")}>
-        {" "}
-        Toggle Custom Sidebar{" "}
+
```
Comment (dwelle) [inline dev-docs/docs/@excalidraw/excalidraw/api/props/render-props.mdx:108]: remove line

File: dev-docs/docs/@excalidraw/excalidraw/api/props/render-props.mdx [hunk 4]
```diff
@@ -55,8 +55,8 @@ function App() {
       <Excalidraw
         renderCustomStats={() => (
           <p style={{ color: "#70b1ec", fontWeight: "bold" }}>
-            {" "}
-            Dummy stats will be shown here{" "}
+
```
Comment (dwelle) [inline dev-docs/docs/@excalidraw/excalidraw/api/props/render-props.mdx:58]: remove line

File: dev-docs/docs/@excalidraw/excalidraw/api/props/render-props.mdx [hunk 5]
```diff
@@ -29,8 +29,8 @@ function App() {
               }}
               onClick={() => window.alert("This is dummy top right UI")}
             >
-              {" "}
-              Click me{" "}
+
```
Comment (dwelle) [inline dev-docs/docs/@excalidraw/excalidraw/api/props/render-props.mdx:32]: remove line

**Comment by dwelle** [comment]
Hey @mohammadamin16. Thanks for the PR.

Some spaces are indeed redundant, but some were needed. I have manually only checked the app-related spaces, but even in the docs there seems to be cases where the space is intentional. Can you re-verify by running the docs locally (`cd dev-docs && yarn start`)?

**Comment by mohammadamin16** [comment]
Hi @dwelle. thanks for your comments. I made the changes and also checked the docs to make sure they're alright .

**Review by dwelle — APPROVED** [review]

**Comment by dwelle** [comment]
thanks @mohammadamin16!

**Pull request merged**